omsApp/src/drvMAXv.cc

```diff
@@ -609,7 +609,7 @@ errorexit:      errMessage(-1, "Invalid device directive");
 static int send_recv_mess(int card, const char * command, const char *axis, char *buf, int nMessages) {
     int retval;
 
-    if (!epicsMutexTryLock(MUTEX(card))) {
+    if (epicsMutexTryLock(MUTEX(card)) != epicsMutexLockOK) {
         Debug(1, "send_recv_mess: waiting for mutex\n");
         epicsMutexLock(MUTEX(card));
```

Comment:
One more set of suggested changes: It occurs to me that the three `epicsMutexLock()` calls should probably use `epicsMutexMustLock()` instead since this code doesn't look at the return value, and the Must version will report an error if there is ever a problem with the mutex.